draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -119,6 +119,33 @@ Below is an example of how an OAuth 2.0 Authorization Server Metadata might incl
 }
 ~~~~
 
+## Nonce Issuer Metadata
+
+The Nonce Issuers that uses the nonce endpoint MUST include in their entity metadata the parameters:
+
+ - **nonce_endpoint**: REQUIRED. This MUST be an HTTPS URL specifying the endpoint where clients can request a Nonce. This parameter is crucial for enabling clients to obtain the necessary Nonce to prevent replay attacks and ensure secure communication.
+
+Below a non-normative example of Nonce Issuer Metadata :
```

Comment:
not sure about this, I would suggest to use a non-noramtive example of an OAuth 2.0 Authorization Server